sdk/turing/router/config/router_config.py

```diff
@@ -230,3 +233,10 @@ def _verify_no_duplicate_routes(self):
             raise turing.router.config.route.DuplicateRouteException(
                 f"Routes with duplicate ids are specified for this traffic rule. Duplicate id: {most_common_route_id}"
             )
+
+    def to_dict(self):
+        att_dict = {}
+        for m in inspect.getmembers(self):
+            if not inspect.ismethod(m[1]) and not m[0].startswith('_'):
```

Comment:
Would be nice to know what m[1] is and m[0] is without looking at the documentation. In this case m[0] is name and m[1] is value. Naming the [magic numbers](https://en.wikipedia.org/wiki/Magic_number_(programming)) makes code easier to read. 

Consider this alternative:

```python
# This is better outside of the function
NAME_INDEX = 0
VALUE_INDEX = 1
if not inspect.ismethod(m[VALUE_INDEX]) and not m[NAME_INDEX].startswith('_'):
    att_dict[m[NAME_INDEX]] = m[VALUE_INDEX]
```

It's much easier to read this.

Reply:
Thanks for the tip; I'll take up that suggestion and put that in :)